--- origsrc/libgpg-error-1.18/src/Makefile.am	2014-10-25 07:42:43.000000000 -0500
+++ src/libgpg-error-1.18/src/Makefile.am	2015-03-19 17:20:39.568407300 -0500
@@ -272,7 +272,7 @@ else
 pre_mkheader_cmds = :
 parts_of_gpg_error_h += ./lock-obj-pub.native.h
 
-./lock-obj-pub.native.h: Makefile gen-posix-lock-obj posix-lock-obj.h
+./lock-obj-pub.native.h: Makefile gen-posix-lock-obj$(EXEEXT) posix-lock-obj.h
 	./gen-posix-lock-obj >$@
 endif
 
