--- origsrc/libgpg-error-1.6/src/Makefile.am	2007-10-29 06:45:05.000000000 -0500
+++ src/libgpg-error-1.6/src/Makefile.am	2007-12-01 20:56:21.312500000 -0600
@@ -61,7 +61,6 @@
 
 gpg_error_res = versioninfo.o
 gpg_error_res_ldflag = -Wl,$(gpg_error_res)
-no_undefined = -no-undefined
 export_symbols = -export-symbols $(srcdir)/gpg-error.def
 
 install-def-file:
@@ -77,7 +76,6 @@
 arch_sources =
 gpg_error_res =
 gpg_error_res_ldflag =
-no_undefined =
 export_symbols =
 install-def-file:
 uninstall-def-file:
@@ -89,7 +87,7 @@
 
 libgpg_error_la_LDFLAGS = $(gpg_error_res_ldflag) -version-info \
   @LIBGPG_ERROR_LT_CURRENT@:@LIBGPG_ERROR_LT_REVISION@:@LIBGPG_ERROR_LT_AGE@ \
-  $(no_undefined) $(export_symbols)
+  -no-undefined $(export_symbols)
 
 libgpg_error_la_SOURCES = gpg-error.h gettext.h $(arch_sources) \
 	init.c strsource.c strerror.c code-to-errno.c code-from-errno.c
@@ -99,7 +97,7 @@
 
 gpg_error_SOURCES = strsource-sym.c strerror-sym.c gpg-error.c $(arch_sources)
 gpg_error_CPPFLAGS = -DLOCALEDIR=\"$(localedir)\"
-gpg_error_LDADD = ./libgpg-error.la @LTLIBINTL@
+gpg_error_LDADD = libgpg-error.la @LTLIBINTL@
 
 err-sources.h: Makefile mkstrtable.awk err-sources.h.in
 	$(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=3 \
